### DISCUSSION TITLE
Can we dynamically select different resolutions?

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#200] [General]
It would be great to be able to enter a new resolution while running.  Maybe like the !keycode entry.  We could use @resolution.

**Comment by GerryDazoo** [comment]
This WAS an option awhile ago. I ended up removing it because changing resolutions on the fly wasn't that useful because of all the other configuration changes needed to stream the new resolution reliably. Ie. VIdeoBandwidth/FrameRate etc.
For now just make another "slingbox" with the required configuration and start a new stream.

**Reply by mdehrlich** [reply]
ok.  I set up a very low res config.  Res 11 is 256 X 192.  Bandwidth shown in VLC is only 450kb max.

**Reply by GerryDazoo** [reply]
don't forget to send the VIdeobandwidth to a much lower value.
